float64 fix_x
std_msgs/String user
---
bool success
